src/ucp/core/ucp_worker.c

```diff
@@ -2056,6 +2056,12 @@ void ucp_worker_create_vfs(ucp_context_h context, ucp_worker_h worker)
     ucs_vfs_obj_add_ro_file(worker, ucp_worker_vfs_show_primitive,
                             &worker->keepalive.round_count, UCS_VFS_TYPE_SIZET,
                             "keepalive/round_count");
+    ucs_vfs_obj_add_ro_file(worker, ucp_worker_vfs_show_primitive,
+                            &worker->num_ep_creations, UCS_VFS_TYPE_ULONG,
```

Comment:
maybe worth adding num of ep_destroy?

Reply:
I'm going to add ep destroy counters in separate PR after clarification of requirements.

Reply:
Ok